lib/donationButtons/ko-fiButton.dart

```diff
@@ -3,7 +3,7 @@ import 'package:simple_icons/simple_icons.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
 ///The 4 Official Kofi Button Colors
-enum KofiColor { Blue, Black, Orange, Red }
+enum KofiColor { Blue, Black, Orange, Red, Grey }
```

Comment:
If you implement the Grey color here as well, I think we should also implement the input blocking?

Reply:
Yes, and I have implemented it in second commit -  148fc20b6239111cf45d3291d5961a14c4fae5c5
Or what do you mean?

Reply:
Ou, yes, I did not add it. Added in my last commit, please check.